### PR TITLE
Add a blog post about GHC23 OSD recap

### PR DESCRIPTION
**Pull request by skyllarr**
https://skyllarr.github.io/wildfly-elytron/blog/ghc23-osd-recap/

### CODE REVIEW

File: _posts/2023-10-19-ghc23-osd-recap.adoc [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+image:
+  path: https://wildfly-security.github.io/wildfly-elytron/assets/images/posts/OpenSourceDay2023.png
+---
+
+[.banner]
+image::https://wildfly-security.github.io/wildfly-elytron/assets/images/posts/OpenSourceDay2023.png[align="left"]
+
+Last month, the https://wildfly-security.github.io/wildfly-elytron/[WildFly Elytron] team had the great pleasure
+of participating in https://ghc.anitab.org/awards-programs/open-source-day/[Open Source Day],
+an all-day hackathon that took place as part of the 2023 https://ghc.anitab.org/[Grace Hopper Celebration]. This was the third year that the WildFly Elytron team has participated in this event. It is always an exciting opportunity for us to meet and help others get started with their open source contributions.
```
Comment (fjuma) [inline _posts/2023-10-19-ghc23-osd-recap.adoc]: s/third year/fourth time

There were two Open Source Days the first year :)

File: _posts/2023-10-19-ghc23-osd-recap.adoc [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+== Preparation
+
+Prior to the day of the event, we created many jira issues and triaged existing ones that would be suitable
+for new contributors to our project. We tried to make sure that the descriptions of these issues contained a lot of
+detail and helpful pointers for getting started. We added the label https://issues.redhat.com/issues/?filter=12364234["good-first-issue"] to the appropriate ones.
+
+We also made sure that our https://wildfly-security.github.io/wildfly-elytron/guides/[setup guides] for different environment are still up to date. We also created https://www.youtube.com/playlist?list=PL6SPIctwCod99nvY6faotopjnEEp71I-B[YouTube videos] on the WildFly channel to let people know that we were participating this event and to invite them to join our WildFly Elytron session.
```
Comment (fjuma) [inline _posts/2023-10-19-ghc23-osd-recap.adoc]: s/are still/were still

Comment (fjuma) [reply _posts/2023-10-19-ghc23-osd-recap.adoc]: s/participating this event/participating in this event

File: _posts/2023-10-19-ghc23-osd-recap.adoc [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+After the motivational opening speeches from the Grace Hopper Conference speakers, the participants chose the Zoom Room of the project they wanted to participate in. At the beginning of the session, we gave a presentation about the Elytron project and introduced ourselves. This year we were very happy to have our former interns, Sonia Zaldana Calles and Ashley Abdel-Sayed, join our project as mentors. We also covered some general information on how to contribute to our project. Then it was time to start coding!
+
+During the hackathon, WildFly Elytron team and mentors helped participants set up their development environment, answered questions about git and the process of submitting and updating pull requests, helped participants choose and work on their problems, and gave feedback on their pull requests.
+
+The hackaton was split into 2 parts, before and after lunch. Many of the OSD attendees switched projects for the afternoon session, so we got some new attendees. The opening speeches and the presentation about our project were part of the afternoon session as well.
```
Comment (fjuma) [inline _posts/2023-10-19-ghc23-osd-recap.adoc]: s/hackaton/hackathon

File: _posts/2023-10-19-ghc23-osd-recap.adoc (hunk 2 above)
Comment (skyllarr) [reply _posts/2023-10-19-ghc23-osd-recap.adoc]: @fjuma Fixed

File: _posts/2023-10-19-ghc23-osd-recap.adoc (hunk 3 above)
Comment (skyllarr) [reply _posts/2023-10-19-ghc23-osd-recap.adoc]: @fjuma Fixed

**Comment by skyllarr** [comment]
@fjuma I updated this PR and added some minor updates

**Review by fjuma — APPROVED** [review]
Thanks very much @Skyllarr for this awesome post!

**Pull request merged**